src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -227,32 +228,32 @@ public boolean isDependentArgumentAllowed(final Class<?> predecessorClass) {
     @Override
     public void validateArguments() {
```

Comment:
Maybe this could be encapsulated in the argument collection as the default implementation, allowing user-specific arguments to contain extra-validation if required...